src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -439,7 +440,6 @@ const uintx max_uintx = (uintx)-1;
 
 typedef unsigned int uint;   NEEDS_CLEANUP
```

Comment:
Maybe, it is because the `uint` is already defined within the guarded lines 149-154: 
```C++
#ifdef __USE_MISC
/* Old compatibility names for C types.  */
typedef unsigned long int ulong;
typedef unsigned short int ushort;
typedef unsigned int uint;
#endif
```

Reply:
I was curious and found it in /usr/include/x86_64-linux-gnu/sys/types.h on ubuntu. I guess it needs cleanup because it might not be defined with USE_MISC?  There are too many int type names to choose from for int.  Why does it say "Old compatibility names for C types."? Should we even be using uint ? I don't see it in https://en.cppreference.com/w/cpp/language/types - should we prefer "unsigned"?

Reply:
This NEEDS_CLEANUP block shouldn't be part of this change.